docs/node/daemon/config/settings.mdx

```diff
@@ -12,46 +12,58 @@ This section provides a detailed overview of the operational parameters availabl
+   - `admin_notls`: Toggle TLS on the admin server. Automatically disabled for unix socket or loopback listen addresses.
+   
+   - `pg_db_host`: PostgreSQL database host (UNIX socket path or IP address with no port). The default is `127.0.0.1` (localhost).
 
    - `pg_db_port`: PostgreSQL database port (may be omitted for UNIX socket hosts). The default is `5432`.
 
    - `pg_db_user`: PostgreSQL database user (should be a "superuser"). The default is `kwild`.
 
-   - `pg_db_pass`: PostgreSQL database pass (may be omitted for some pg_hba.conf configurations).
+   - `pg_db_pass`: PostgreSQL database pass (may be omitted for some pg_hba.conf configurations). No default.
 
    - `pg_db_name`: PostgreSQL database name (override database name). The default is `kwild`.
 
    - `private_key_path`: This variable specifies the path for node's private key. If left unspecified, it defaults to `./private_key` relative to the root directory.
 
+   - `profile_mode`: This variable sets the kwild profiling module. Options are `http`, `cpu`, `mem`, `mutex`, and `block`. No default.
+
+   - `profile_file`: The kwild profile output file path. Not enabled by default.
+
+   - `rpc_timeout`: The timeout for the JSON-RPC server. Default is 45s. 
+
+   - `db_read_timeout`: The timeout for database reads initiated by RPC requests. Default is 5s. 
+
    - `extension_endpoints`: This variable specifies the endpoints for extensions, separated by commas.
 
    - `tls_cert_file`: This variable specifies the file path containing a certificate to enable TLS on the JSON-RPC server. This path can be either absolute or relative to the specified root directory. If unspecified, the JSON-RPC server's communications remain unencrypted.
 
    - `tls_key_file`: This variable specifies the file path containing the corresponding private key that enables TLS on the JSON-RPC server. Like the `tls_cert_file`, this path can be either absolute or relative to the specified root directory. If unspecified, the JSON-RPC server's communications remain unencrypted.
 
-   - `hostname`: This variable sets the Kwil JSON-RPC server's hostname.
+   - `hostname`: This variable sets the Kwil JSON-RPC server's hostname. No default.
 
-   - `genesis_state`: This variable specifies the path to the snapshot file to be used for the initial state of the node. If the node is configured to start with initial state and this variable is not set, the node will fail to start.
+   - `genesis_state`: This variable specifies the path to the snapshot file to be used for the initial state of the node. If the node is configured to start with initial state and this variable is not set, the node will fail to start. No default.
 
    ### [app.snapshots]
 
-      This section contains configurations for managing the snapshots of the kwild database.
+      This section contains configurations for managing the snapshots of the kwild database. Snapshot variables do not have a default value.
```

Comment:
@charithabandi is it right that these don't have defaults?  I believe these are set:

```go
			Snapshots: SnapshotConfig{
				Enabled:         false,
				RecurringHeight: 14400, // 1 day at 6s block time
				MaxSnapshots:    3,
				SnapshotDir:     SnapshotDirName, // "snapshots"
			},
```